### ISSUE TITLE
Remove voter biase

### ISSUE THREAD

**Issue by BEllis** [openpolitics/manifesto#635]
I think any political voting system such as the Open Manifesto should really remove all barriers to voting on proposals so as to remove bias (while obviously still verifying the authenticity of votes).

As it stands I see three big problems,
1. only those who raise successful proposals can vote, this means only those who make proposals with similar political view-points as the current voter-base will be accepted and so re-enforce the poltical bias of the voter-base.
2. I do not see many members of the older generations voting via an online portal, this produces a biased vote towards younger/non-technical political views.
3. Fear of blackmail, etc. Only those who believe in being politically open will vote, while those of a more conservative political view will not, this puts the bias on those who are less concerned about security and personal well being.

I'm not sure how best to address the above issues, but below are some thoughts,
1. Allow any "verified" user to vote would be a good start including verification of their home address (so those outside the regional manifesto cannot influence it?). The issue for me here is trust in the organisation administrating it, I'm not sure I'd be happy sending a copy of my passport or home address to a relatively un-trusted organisation. This would hopefully be alleviated over time as the openpolitics system becomes more widely adopted and therefore trusted to keep my personal information  safe.

2. Provide the ability to subscribe to a paper-based voting mechanism (such as a monthly mailshot, with the latest changes, latest version of the manifesto and a ballot paper with recent proposals and a yay or nay or abstain box next to each?)

3. Allow for opt-in/out anonymous voting but keep a "voters register" much like the electoral commission in the UK?

**Comment by Floppy** [comment]
Thanks! Good points here. I'm looking for ways to support alternative methods of deciding who can vote, so these are all great thoughts we should take into account!

**Comment by Floppy** [comment]
To address the problems you raise (not to dismiss, but to explain my own thinking):

1. "only those who make proposals with similar political view-points as the current voter-base will be accepted" - I tend to think of this as a feature rather than a problem, at the moment. Because the aim of the project is to build a coherent manifesto, not to run a direct democracy platform, a certain coherence of vision (or an echo chamber) is perhaps desirable. But if the idea (perhaps in future) was to be fully democratic and open to all, obviously this would be a problem, as you point out.

2. bias towards those online - agreed, it's a problem, though I've been surprised at the diversity of input we *have* seen. Again it would be an even more significant problem for a direct democracy application of this idea.

3. Good point, although at the moment anonymity is possible (to an extent) via a github account not linked to anything else, I should be careful to make sure that remains possible in future, and becomes a core design goal.

**Comment by BEllis** [comment]
Thanks for the reply.

My thinking is if "Something New" wanted to win more votes, aiming to be populist party would probably work best, so I suppose it depends on the goals of Something New. I do agree with the comment regarding keeping the manifesto coherent but I'd hope this could be achieved via a direct democracy if suitable safe-guards are in place.

re: anonymity, I was thinking more on a vote-by-vote basis but I think what you've suggested is equivalent to the way the Electoral Registrar works. My thinking is if there is a controversial topic, I may want to vote but not have my name against it in case of unwanted recompense.

**Comment by BEllis** [comment]
Just thinking some more, re: anonymity and github accounts, if there is nothing stopping me creating a random github account, then I'd assume there is nothing stopping me from voting multiple times by setting up several accounts?

**Comment by Floppy** [comment]
yeah, that's the corollary to the anonymity... definitely a concern.
